sys/net/iflib.c

```diff
@@ -1650,8 +1651,12 @@ iflib_fast_intr_ctx(void *arg)
 			return (result);
 	}
 
-	if (gtask->gt_taskqueue != NULL)
-		GROUPTASK_ENQUEUE(gtask);
+	/*
```

Comment:
The driver should probably be doing bus_teardown_intr() first before draining the taskqueue to close this race instead.

Reply:
So taskqueue_free in iflib_device_deregister should go after IFDI_DETACH, which tears down MSI-X, and iflib_free_intr_mem, which takes care of MSI and legacy interrupts. Kevin Bowling merged first patch from this PR. Do you want me to update it anyway or open a new one?